mazerunner-scenarios/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.4.1)
+
+add_library(entrypoint SHARED
+             src/main/cpp/entrypoint.cpp
+             src/main/cpp/bugsnags.cpp)
+
+#FIXME determine how best to link against bugsnag-ndk library in mazerunner module
```

Comment:
This will be addressed in a future PR. Any mazerunner scenarios which require direct use of bugsnag's C API have not been ported yet. e.g.

```
bugsnag_leave_breadcrumb() // etc
```